include/openthread/platform/settings.h

```diff
@@ -51,6 +51,29 @@ extern "C" {
  *
  */
 
+/**
+ * This enumeration defines the keys of settings.
+ *
+ * Note: When adding a new setings key, if the settings corresponding to the key contains security sensitive
+ *       information, the developer MUST add the key to the array `kCriticalKeys`.
+ *
+ */
+enum
+{
+    OT_SETTINGS_KEY_ACTIVE_DATASET  = 0x0001, ///< Active Operational Dataset
```

Comment:
```suggestion
    OT_SETTINGS_KEY_ACTIVE_DATASET  = 0x0001, ///< Active Operational Dataset.
```
And other lines at below.